examples/basic-examples/test/e2e/integration/smoke/middleware.feature

```diff
@@ -0,0 +1,13 @@
+Feature: Custom middleware
+
+  Custom middleware should work
```

Comment:
As a developer I want to be able to add custom components to my ripple instance through middleware